### PR TITLE
Edited new AWS container doc for global consistency 

### PR DESCRIPTION
**Pull request by ghost**
Reviewed per Pierre's request to create consistency: 

* numbered the steps
* put borders around images
* defined images directory at beginning of document (as opposed to prefixing it to each image file)  

Changes to Kaui doc was only to add a link to the Kaui document outline

### CODE REVIEW

**Comment by ghost** [comment]
NOTE: The image files will not display the borders until the images are updated on killbill:v3. 

Also, @jdmooney, for the step in which they need to copy the contents of a .json file, it's not clear to me where the contents should be copied. Can you clarify?

**Comment by jdmooney** [comment]
I have looked through the serverless option document. I am not at all familiar with this option or with ECS. I tried to follow the instructions in the document, but I cannot find the right Marketplace page, if there is one, and there is no link. Without this I could not create a cluster.
I assume that selecting the JSON option would open a box into which the given file can be pasted. But I cannot verify this.

File: userguide/tutorials/aws-container.adoc [hunk 1]
```diff
@@ -8,48 +15,68 @@ While we recommend using our https://docs.killbill.io/latest/aws-cf.html[Cloudfo
...
 
-image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-cluster-fargate.png[align=center]
+[start=4]
+. Enter a cluster name:
 
-Enter a name:
+image:ecs-cluster-name.png[align=center]
 
-image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-cluster-name.png[align=center]
+If these above steps are succesfful, AWS displays a success screen:
```
Comment (pierre) [inline userguide/tutorials/aws-container.adoc:35]: `ff`

**Comment by ghost** [comment]
I'm closing this because it has other commits tangled up in it.

**Pull request closed without merging**